docs/aws/README.md

```diff
@@ -37,7 +37,7 @@ aws cloudformation deploy \
 Karpenter relies on [cert-manager](https://github.com/jetstack/cert-manager) for Webhook TLS certificates.
 
 ```bash
-./hack/quick-install.sh
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/awslabs/karpenter/v0.2.1/hack/quick-install.sh)"
```

Comment:
Is it possible to change this when we do `make release`?

Reply:
Though to automate, but not impossible. I wish we could refer to git tags in READMEs, but not possible in 2021 :(